schema.ddl

```diff
@@ -1,48 +1,115 @@
+/*
+ * Machine
+ */
 CREATE TABLE IF NOT EXISTS machine (
-	uuid 			TEXT PRIMARY KEY,
-	id 				TEXT NOT NULL UNIQUE,
-	nonce 			TEXT,
-	series 			TEXT,
-	container_type	TEXT
+	uuid						TEXT PRIMARY KEY,
+	id							TEXT NOT NULL UNIQUE,
+	nonce						TEXT,
+	series						TEXT,
+	container_type				TEXT,
+	life						INT,
+	password_hash				TEXT,
+	clean						BOOLEAN,
+	force_destroyed				BOOLEAN,
+	preferred_public_address	TEXT,
+	preferred_private_address	TEXT,
+	supported_containers_known	BOOLEAN,
+	placement					TEXT,
+	stop_mongo_until_version	TEXT,
+	agent_started_at			TEXT,
+	hostname					TEXT
+	-- principles
+	-- tools
+	-- jobs
+	-- addresses
+	-- machine_addresses
+	-- supported_containers
 );
 
 CREATE UNIQUE INDEX IF NOT EXISTS idx_machine_id
 ON machine (id);
 
+/*
+ * File system
+ */
 CREATE TABLE IF NOT EXISTS filesystem (
-	uuid 			TEXT PRIMARY KEY,
-	id 				TEXT NOT NULL UNIQUE,
-	releasing		BOOLEAN
+	uuid				TEXT PRIMARY KEY,
+	id					TEXT NOT NULL UNIQUE,
+	releasing			BOOLEAN,
+	life				INT,
+	attachment_count	INT
```

Comment:
We won't need these counts any longer. They are used in transaction assert ops in lieu of joins.